roles/dmi/tasks/main.yml

```diff
@@ -55,7 +55,7 @@
 
 - name: add dmi.processor-family to dictionary
   set_fact:
-    dmi: "{{ dmi|default({}) | combine({ item: dmi_processor_family['stdout'] | trim | default('error') }) }}"
+    dmi: "{{ dmi|default({}) | combine({ item: dmi_processor_family['stdout'] | trim | default('error') if have_dmidecode else 'N/A (dmidecode not found)' }) }}"
```

Comment:
For future maintainability, you may consider just creating a variable for this error message since you are typing it verbatim several times.  

i.e.  `no_dmi: 'N/A (dmidecode not found)'`

I'm sure there is some fancier way to do this with jinja2 as well to just format your error message to include the program name, but can't think of it off the top of my head.

Reply:
@kdelee  This is a good idea, but I've been looking through the Ansible documentation and I don't know how to define a variable like this in a way that doesn't involve a lot of overhead. I'm going to skip this for now, but please let me know if you figure out a way.